printstraat/frontend/projects/valtimo/printstraat/src/lib/components/printstraat-configuration/printstraat-plugin-configuration.component.scss

```diff
@@ -0,0 +1,15 @@
+/*!
```

Comment:
deze hele klasse mag je weggooien

Reply:
Zonder die klasse gaat hij stuk en start die niet op dus deze moet er helaas wel zijn

Reply:
Nee, dit kan. Zie opmerking in volgende .ts file

Reply:
dat heb ik geprobeerd maar dan blijft die stuk gaan